### PR TITLE
[BIA-893] DO NOT MERGE. Investigate how we can run a docker container where we can execute AMT tests - SQL Server

### PR DESCRIPTION
**Pull request by jleiva-gap**
- Add a MSSQL 2017 Express  docker database in the yaml file.
- Add secrets to store database credentials.
- Disable (for testing purposes) Postgres test.
- Create a fork to test the workflow.
- Run Integration test.

image

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -14,12 +14,39 @@ on:
...
+  POSTGRES_POOLING: "false"
 
 jobs:
   build:
-    runs-on: windows-latest
+    #runs-on: Windows-latest
+    runs-on: Ubuntu-latest
 
     steps:
+      - name: Start windows container for SQLServer 2017 Express
```
Comment (stephenfuqua) [inline .github/workflows/build.yml:42]: This is actually a Linux-based container.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -32,4 +59,17 @@ jobs:
             ${{ runner.os }}-nuget-
     
       - name: Building AMT
-        run: .\build.ps1 build -Configuration release -Version ${{ env.BUILD_VERSION }} -BuildCounter ${{ github.run_number }}
\ No newline at end of file
+        run: |
+          .\build.ps1 build -Configuration ${{ env.CONFIGURATION }} -Version ${{ env.BUILD_VERSION }} -BuildCounter ${{ github.run_number }}
+        shell: pwsh
```
Comment (stephenfuqua) [inline .github/workflows/build.yml:64]: Is this actually working in PowerShell Core? That would be sweet. I don't think the other teams have started testing with PSHell Core yet.

File: .github/workflows/build.yml (hunk 1 above)
Comment (jleiva-gap) [reply .github/workflows/build.yml:42]: Yes, I changed the script to use Ubuntu instead of Windows.

File: .github/workflows/build.yml (hunk 2 above)
Comment (jleiva-gap) [reply .github/workflows/build.yml:64]: Yes, it is working, I'm using Ubuntu. If we don't include the last line it will failed. I ran the build and tests.

**Comment by DavidJGapCR** [comment]
IMPORTANT! We are not merging this, since [BIA-893] (https://tracker.ed-fi.org/browse/BIA-893) is just a spike ticket to -at a high level- learn about docker and its integration with GitHub Actions. The actual implementation is part of another ticket. The screenshots prove, and I have been able to tests on my own that our objective has been accomplished.

**Pull request closed without merging**